kafka_consumer/ci/kafka_consumer.rake

```diff
@@ -0,0 +1,65 @@
+require 'ci/common'
+
+def kafka_consumer_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+namespace :ci do
+  namespace :kafka_consumer do |flavor|
+    task before_install: ['ci:common:before_install']
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements('kafka_consumer/requirements.txt',
+                           "--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      sh %(docker-compose -f #{ENV['TRAVIS_BUILD_DIR']}/kafka_consumer/ci/resources/docker-compose-single-broker.yml up -d)
+      Wait.for 2181
+      Wait.for 9092
+      sleep_for 5
```

Comment:
what are we waiting for here? Could it become a loop waiting on something more precise instead?